lib/kramdown-rfc2629.rb

```diff
@@ -1317,10 +1320,10 @@ def self.bcp_std_ref(t, n)
                   ->(fn, n){ [fn,
                               "https://datatracker.ietf.org/doc/bibxml3/draft-#{n.sub(/\Adraft-/, '')}.xml"] }
                  ],
-        "BCP" => ["bibxml-rfcsubseries", 86400*7, false,
+        "BCP" => ["bibxml-rfcsubseries", KRAMDOWN_REFCACHETTL_RFC, false,
                   ->(fn, n){ Rfc2629::bcp_std_ref("BCP", n) }
                  ],
-        "STD" => ["bibxml-rfcsubseries", 86400*7, false,
+        "STD" => ["bibxml-rfcsubseries", KRAMDOWN_REFCACHETTL_RFC, false,
                   ->(fn, n){ Rfc2629::bcp_std_ref("STD", n) }
                  ],
         "W3C" => "bibxml4",
```

Comment:
@cabo, while David has made this a targeted change, I think that you should consider making the TTL on most of these other reference types much longer.  (Maybe W3C specs are the exception in that they can change faster, but things like NIST specs don't change that often.)

Reply:
Right.  Why is that important?  The current values are based on the observation that a false cache hit is much worse than a false cache hit.  Now the fact that this cache even exists is hammered into our heads by the refilling being very verbose, and that is the problem I planned to work on next.
(See https://github.com/cabo/kramdown-rfc/issues/243 .)

Reply:
My use case is that I'm currently on a train with incredibly slow and flaky Wi-Fi. I'd like to be able to control the values from environment variables so that I can use increase them in such a scenario.

Reply:
I like your change if only for reducing the amount of unmotivated pulled-out-of-thin-air constants in the code.
We don't have a CLI for invalidating cache entries, and the current default lifetimes are way too long if bib.ietf.org finally just fixed one of its bugs and you'd prefer for that fix to make it into your documents... 
(It is a bit like pushing a new revision of kramdown-rfc and having to wait two weeks for that to get usable in CI, cough, cough, except that for that it is even harder to trigger a refresh.)

Reply:
Agreed!

Are there any next steps you'd like me to do towards merging this PR?

Reply:
I'll merge after a bit of reviewing and testing...